### PR TITLE
replace MW2Runner with fornax runner

### PR DESCRIPTION
**Pull request by andreasmuelder**

### CODE REVIEW

File: plugins/com.yakindu.solidity/pom.xml [hunk 1]
```diff
@@ -100,53 +96,37 @@
...
 							</excludes>
 						</fileset>
 					</filesets>
 				</configuration>
 			</plugin>
 			<plugin>
-				<groupId>org.codehaus.mojo</groupId>
-				<artifactId>exec-maven-plugin</artifactId>
-				<version>1.4.0</version>
+				<groupId>org.fornax.toolsupport</groupId>
```
Comment (cdietrich) [inline plugins/com.yakindu.solidity/pom.xml:105]: you know that this one is unmaintained?

Comment (andreasmuelder) [reply plugins/com.yakindu.solidity/pom.xml:105]: Yes I know, but it is working as expected. The official way of generating languages during build get stuck too often.

Comment (cdietrich) [reply plugins/com.yakindu.solidity/pom.xml:105]: that do you mean by "get stuck"? was this ever investigated?

Comment (andreasmuelder) [reply plugins/com.yakindu.solidity/pom.xml:105]: I think we had a discussion for Yakindu/statecharts. The build freezes and gets killed after 10 minutes of no activity. For solidity-tools, we had some dependency issues: 

[ERROR] Failed to execute goal org.codehaus.mojo:exec-maven-plugin:1.4.0:java (mwe2Launcher) on project com.yakindu.solidity: Execution mwe2Launcher of goal org.codehaus.mojo:exec-maven-plugin:1.4.0:java failed: Plugin org.codehaus.mojo:exec-maven-plugin:1.4.0 or one of its dependencies could not be resolved: Failed to collect dependencies at org.codehaus.mojo:exec-maven-plugin:jar:1.4.0 -> org.eclipse.emf:org.eclipse.emf.mwe2.launch:jar:2.11.3 -> org.eclipse.emf:org.eclipse.emf.mwe2.runtime:jar:[2.11.3]: No versions available for org.eclipse.emf:org.eclipse.emf.mwe2.runtime:jar:[2.11.3] within specified range -> [Help 1]

Comment (cdietrich) [reply plugins/com.yakindu.solidity/pom.xml:105]: the 2.11.3 can be easily solved by calling maven with -U

**Pull request merged**